src/scripts/admin/filters/media-text-enhanced.js

```diff
@@ -14,3 +22,71 @@ registerBlockStyle('core/media-text', {
 	name: 'no-clip',
 	label: 'Full Image',
 });
+
```

Comment:
seems like a lot of work to just add a title hover, at least WordPress is sticking to their core principles

Reply:
Agreed.